### PR TITLE
Corrections on html tags

### PR DESCRIPTION
**Pull request by RoyNyaga**

### CODE REVIEW

**Comment by RoyNyaga** [comment]
Please if you want to use the online html validator at https://validator.w3.org/, make sure to upload the html file itself rather than copying the code and pasting in the Validator if not false errors will be raised. We created this branch uniquely for submission of this project so to check out my partners work, look on other branches, the errors where very small for the two of us to commit changes, so one person did the correction while the other gave supplied ideas. Thanks for understanding.

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a Microverse project and consist of producing a clone of Mint.com’s signup page using html and css. To view this project life, you can visit the link https://roynyaga.github.io/microverse_html_form_project/
```
Comment (bolah2009) [inline README.md:1]: `SUGGESTION`

Nice job writing a descriptive `README.md` file for describing what the project does. You can even add other information like live page link, assignment link, collaborators and a screenshot of the webpage. :+1: :star2:

File: debug.log [hunk 2]
```diff
@@ -0,0 +1,117 @@
+[0712/175608.106:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (bolah2009) [inline debug.log:1]: `SUGGESTION`

Consider deleting this file or had to `.gitignore` if you really want to keep it.

**Review by bolah2009 — COMMENTED** [review]
# Nice Job :+1:

**Pull request merged**